### PR TITLE
Add tweet block

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

**Comment by nlisgo** [comment]
It is my understanding that, I will need to adjust the versions of all endpoints that support the new tweet block.

**Comment by thewilkybarkid** [comment]
Technically this should have a version bump. However, it's probably ok to be pragmatic about it and add support everywhere before allowing content to be added containing it.

**Comment by nlisgo** [comment]
> Technically this should have a version bump. However, it's probably ok to be pragmatic about it and add support everywhere before allowing content to be added containing it.

I could deploy the changes to Journal CMS last.

File: src/blocks/tweet.v1.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        title: Tweet ID
+        type: string
+    date:
+        $ref: ../misc/partial-date.v1.yaml
+    text:
+        title: Text
+        type: array
+        items:
+            $ref: paragraph.v1.yaml
+        minItems: 1
```
Comment (thewilkybarkid) [inline src/blocks/tweet.v1.yaml]: Do we have paragraph blocks, or a single string of HTML?

Comment (nlisgo) [reply src/blocks/tweet.v1.yaml]: > Do we have paragraph blocks, or a single string of HTML?

Single string of html.

**Pull request merged**